alibi/explainers/ale.py

```diff
@@ -106,7 +106,20 @@ def explain(self, X: np.ndarray, features: List[int] = None, min_bin_points: int
 
         Returns
         -------
+        Explanation
             An `Explanation` object containing the data and the metadata of the calculated ALE curves.
+            Data-related attributes:
+             - `ale_values` : List[np.ndarray] - a list of arrays of ALE values (one for each feature). Each array \
+             can have multiple columns (if the number of targets is >1 as in classification).
+             - `constant_value` : float - the mean prediction over  (zeroth order effects).
+             - `ale0` : List[np.ndarray] - a list of “centering” values (one for each feature) used by the algorithm \
+             to center the `ale_values` around the expected effect for the feature (i.e. the sum of `ale_values` \
+             and `ale0` will be the uncentered ALE).
+             - `feature_values` : List[np.ndarray] - a list of arrays (one for each feature) of feature values at \
+             which the ALE values were computed.
+             - `feature_names` : np.ndarray - a list of feature names.
+             - `target_names` : np.ndarray - a list of target names.
+             - `feature_deciles` : List[np.ndarray] - a list of arrays (one for each feature) of the feature deciles.
```

Comment:
A quick idea to test - instead of repeating all this information (which is on the methods page) and risk information going out of sync, can we link to the relevant page/section insead? Could you check if a relative link works? You may need to look up hyperlink/relative link syntax in `.rst` files.

Reply:
On second thought this is not a good idea since the library isn't shipped with any external documentation and the docstrings should be fairly self-contained. Let's leave this as is and make a mental note for any changes to the returned `Explanation` to update docs in two places. In the future this would be hopefully superseeded by having `pydantic` explanation types we can link to which would be self-documenting.